src/translations/nb/publicationTypes.json

```diff
@@ -1,20 +1,30 @@
 {
   "Book": "Bokpublikasjon",
   "BookAnthology": "Antologi",
-  "BookMonograph": "Monograf",
-  "Chapter": "Kapittel",
-  "Degree": "Grad",
-  "DegreeBachelor": "Bachelorgrad",
-  "DegreeMaster": "Mastergrad",
-  "DegreePhd": "PhD",
+  "BookMonograph": "Monografi",
+  "ChapterArticle": "Kapittel i antologi",
+  "Degree": "Studentoppgave",
+  "DegreeBachelor": "Bacheloroppgave",
+  "DegreeMaster": "Masteroppgave",
+  "DegreePhd": "Doktoravhandling",
+  "DegreeOther": "Annen studentoppgave",
   "Journal": "Publikasjon i tidsskrift",
-  "JournalArticle": "Artikkel",
+  "JournalArticle": "Artikkel i tidsskrift",
+  "JournalBookReview": "Bokanmeldelse",
+  "JournalCorrigendum": "Korrigendum",
+  "JournalEditorial": "Lederartikkel",
   "JournalLeader": "Leder",
   "JournalLetter": "Brev til redaktør",
   "JournalReview": "Anmeldelse",
   "JournalShortCommunication": "Short communication",
+  "Other": "Annen publikasjon",
+  "OtherFeatureArticle": "Kronikk",
+  "OtherMap": "Kart",
+  "OtherMusical": "Musikktrykk",
+  "OtherPublication": "Annen publikasjon",
```

Comment:
Other og OtherPublication er det samme??

Reply:
jepp 🤷‍♂️

Reply:
hva er poenget? det er jo det samme på engelsk også?

Reply:
å sånn ja, du vil slå de sammen til å bruke samme key?
ikke usannsynlig at de vil ha andre navn i backend da, så da må vi ha to ulike felter for det (dette er jo ikke implementert enda, bare placeholders for oss slik at det er klart når backend har støtte), men kan bruke `OtherPublication: t("Other")`

I praksisk er dette oversetting av alle typene vi skal ha, hvilke verdier de vil få i backend (som blir keys i dette tilfellet) er veldig tentativt, derfor syns jeg det er greit at det er en egen rad. Eks: `ChapterArticle` sommer helt sikkert til å endres til noe ala `BookChapter` når backend flytter kapittel fra å være hovedtype til undertype

Reply:
backend kommer ikke til å flytte kapittel til undertype (men det er egentlig en annen sak da). Tanken er at vi i frontend ikke bør være så avhengig av backend, så de kommer til å flate ut modellen, slik at vi ikke trenger å ha samme typer som de. Og når vi ikke trenger det, så trenger vi hvertfall ikke duplikat av samme type?

Reply:
vi må vel ha samme typer som backend for publikasjon? nå er jo hovedtype (`publicationContext.type`) `Journal`, `Book`, osv, mens undertype (`publicationInstance.type`) er `JournalArticle`, `JournalShortCommunication`, `BookAnthology` osv. Hvis vi ikke har samme typer som backend her vil det ikke være mulig å ha ulik validering basert på typer heller. feks er `JournalArticle` den eneste undertypen av `Journal` som skal kunne være fagfellevurdert.

Denne fila skal nå representere alle typene, og  hovedtypen `Other` er ikke det samme som undertypen `OtherPublication`, men igjen: backend har ikke støtte for disse enda, så det kan være de blir hetende noe annet, men oversettelsen skal fortsatt være lik

Reply:
Det jeg snakket med Rurik om var at vi kun trengte å sende undertyper, så kunne de inferere hvilken hovedtype det hørte til, så vi ikke trenger å ha samme hovedtype som backend -> slik at vi kan lagre Kapittel som undertype av Bok (men så er det egentlig ikke det i modellen). Dette kommer sikkert til å bli diskutert mer etterhvert som disse endringene gjøres da, men jeg mener at vi ikke trenger begge typene i frontend. Men vi kan ta det etterhvert isteden da